### PR TITLE
fix typos and some kind of translation clarification.

### PR DESCRIPTION
**Pull request by aflyhorse**
as title.

### CODE REVIEW

**Comment by KagamiChan** [comment]
非常感谢～
不过未卜先知的话，更像是 battle result prediction 的感觉？

**Comment by aflyhorse** [comment]
outcome 相比 result 更加正式点，一般战役结果喜欢用这个词……不过既然是游戏hmm……
居然年三十还能存活并回复@@

File: assets/data/plugin.json [hunk 1]
```diff
@@ -58,9 +58,9 @@
     "author": "马里酱",
     "link": "https://github.com/malichan",
     "deszh-CN": "远征信息查询 & 成功条件检查",
-    "deszh-TW": "远征信息查询 & 成功条件检查",
-    "desja-JP": "远征信息查询 & 成功条件检查",
-    "desen-US": "远征信息查询 & 成功条件检查",
+    "deszh-TW": "遠征信息查詢 & 成功條件檢查",
+    "desja-JP": "遠征情報お調べと成功条件チェック",
+    "desen-US": "Inquiry & Verify Expedition requirement",
```
Comment (KagamiChan) [inline assets/data/plugin.json]: `Expedition requirement inquiry and validation` might be better

Comment (aflyhorse) [inline assets/data/plugin.json]: Should i retain the & symbol in every translation? :)

Comment (KagamiChan) [inline assets/data/plugin.json]: Feel free to use `&`, I just forgot it

**Comment by aflyhorse** [comment]
There's also a missing KR translation at the bottom, but I'm afraid I couldn't lend a hand. :(

**Pull request merged**